Formula/bitcoin.rb

```diff
@@ -53,7 +42,7 @@ def install
                           "--with-boost-libdir=#{Formula["boost"].opt_lib}",
                           "--prefix=#{prefix}"
     system "make", "install"
-    pkgshare.install "share/rpcuser"
+    pkgshare.install "share/rpcauth"
```

Comment:
[Here's the source for this change](https://bitcoin.org/en/release/v0.16.0#renamed-script-for-creating-json-rpc-credentials) for posterity 😄